workday.sgnl.yaml

```diff
@@ -0,0 +1,92 @@
+description: "Workday as a System of Record"
+address: "wd2-impl-services1.workday.com" 
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: Workday-1.0.0
+adapterConfig: ewogICAgImFwaVZlcnNpb24iOiAidjEiLAogICAgIm9yZ2FuaXphdGlvbklkIjogInt7SW5wdXQgUmVxdWlyZWR9fSIKfQ==
+auth:
+  - bearer:
+      authToken: "Bearer {{Input Required}}"
+entities:
+  Worker:
+    displayName: Worker
+    externalId: allWorkers # Name of table in Workday
+    description: Worker entity in Workday
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workerId
+        externalId: $.worker.id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: workerDescriptor
+        externalId: $.worker.descriptor
+        type: String
+      - name: employeeID
+        externalId: employeeID
+        type: String
+      - name: workerActive
+        externalId: workerActive
+        type: Bool
+      - name: managementLevelDescriptor
+        externalId: $.managementLevel.descriptor
+        type: String
+      - name: managementLevelId
+        externalId: $.managementLevel.id
+        type: String
+      - name: companyDescriptor
+        externalId: $.company.descriptor
+        type: String
+      - name: companyId
+        externalId: $.company.id
+        type: String
+      - name: genderDescriptor
+        externalId: $.gender.descriptor
+        type: String
+      - name: genderId
+        externalId: $.gender.id
+        type: String
+      - name: hireDate
+        externalId: hireDate
+        type: DateTime
+      - name: FTE
+        externalId: FTE
+        type: String
```

Comment:
Maybe index this attribute?

cc: @marc-sgnl for inputs on attributes to index based on the usecase.

Reply:
added index

Reply:
just added one comment that I think is worth indexing